### PR TITLE
ceph-ansible-pipeline: Skip *lvm* scenarios if merging into stable-3.0

### PR DESCRIPTION
**Pull request by djgalloway**
Signed-off-by: David Galloway <dgallowa@redhat.com>

### CODE REVIEW

File: ceph-ansible-pipeline/config/definitions/ceph-ansible-pipeline.yml [hunk 1]
```diff
@@ -59,23 +59,54 @@
...
-                  - name: 'ceph-ansible-prs-luminous-ansible2.4-purge_filestore_osds_non_container'
-                    current-parameters: true
-                  - name: 'ceph-ansible-prs-luminous-ansible2.4-purge_lvm_osds'
-                    current-parameters: true
+            # If PR is merging into stable-3.0 branch HEAD, run all but LVM scenarios
+            - conditional-step:
+                condition-kind: shell
+                condition-command: |
+                  #!/bin/bash
+                  git show-branch ${sha1} | grep -q $(git ls-remote https://github.com/ceph/ceph-ansible.git stable-3.0 | awk '{ print $1 }')
```
Comment (leseb) [inline ceph-ansible-pipeline/config/definitions/ceph-ansible-pipeline.yml]: Out of curiosity, where is the sha1 coming from?
Why don't we take `git rev-parse --abbrev-ref HEAD` and then compare it to `stable-3.0`?

Sounds easier to me.

Comment (djgalloway) [reply ceph-ansible-pipeline/config/definitions/ceph-ansible-pipeline.yml]: `${sha1}` comes from the github trigger.  It gets set to `refs/remotes/origin/pr/2811/merge` as an example.

Comment (djgalloway) [reply ceph-ansible-pipeline/config/definitions/ceph-ansible-pipeline.yml]: Example
```
$ git show-branch origin/pr/2633/merge 
[origin/pr/2633/merge] Merge e8c3745eaf18533f0871b7a629093029359db10e into 7e068d0b93ccc2d25c3a8f07454c534c8a26a758
```

Comment (leseb) [reply ceph-ansible-pipeline/config/definitions/ceph-ansible-pipeline.yml]: Thanks!

**Pull request merged**